uk.co.compucorp.civicrm.tasksassignments/gulpfile.js

```diff
@@ -17,7 +20,20 @@ gulp.task('sass', function (done) {
   });
 });
 
-gulp.task('requirejs-bundle', function (done) {
+gulp.task('js-crm', function () {
+  return gulp.src([
+    'js/src/crm-tasks-workflows/modules/*.js',
+    'js/src/crm-tasks-workflows/controllers/*.js',
+    'js/src/crm-tasks-workflows.js'
+  ])
+    .pipe(sourcemaps.init())
+    .pipe(concat('crm-tasks-workflows.min.js'))
+    .pipe(uglify())
+    .pipe(sourcemaps.write())
+    .pipe(gulp.dest('js/dist/'));
+});
+
+gulp.task('js-requirejs-bundle', function (done) {
```

Comment:
why was this renamed? shouldn't we follow the same task name as in CiviHR? anyways, js-requirejs-bundle is redundant.